torch_em/data/datasets/light_microscopy/plantseg.py

```diff
@@ -56,6 +83,36 @@
 # NATIVE_RESOLUTION = (0.235, 0.075, 0.075)
 
 
+def _fix_inconsistent_volumes(data_path, name, split):
+    file_paths = glob(os.path.join(data_path, "*.h5"))
+    if name not in ["root", "ovules"] and split not in ["train", "val"]:
+        return
+
+    for vol_path in tqdm(file_paths, desc="Fixing inconsistencies in volumes"):
+        fname = os.path.basename(vol_path)
+
+        # avoid duplicated volumes in 'train' and 'test'.
+        if fname == "Movie1_t00045_crop_gt.h5" and (name == "plantseg" and split == "train"):
```

Comment:
I don't see how `name == "plantseg"` can occur here. The only valid names are `"root", "ovules", "nuclei"`. Do you mean `"root"`?

Reply:
Oops, yes. That's a mistake from my side. Thanks for spotting!